src/main/java/fact/datacorrection/InterpolateTimeSeries.java

```diff
@@ -36,6 +36,12 @@
     @Parameter(required = false, description = "The minimum number of neighboring pixels required for interpolation", defaultValue = "3")
     public int minPixelToInterpolate = 3;
 
+    @Parameter(required = false, description = "The key for the resulting badPixelSet.")
+    public String badPixelKey = "badPixels";
```

Comment:
Why is this an xml Parameter?

Reply:
I use the processor twice, so i added this in case someone want to preserve the output and not override the key with the second processor. I think it is cleaner this way, but I can remove it.